llmfoundry/registry.py

```diff
@@ -206,6 +206,17 @@
     description=_metrics_description,
 )
 
+_icl_dataloaders_description = (
```

Comment:
I guess this is actually datasets that are registered so should probably be called icl_datasets?